gridcapa-swe-runner-app/src/main/java/com/farao_community/farao/swe/runner/app/dichotomy/RaoValidator.java

```diff
@@ -81,6 +84,13 @@ public DichotomyStepResult<SweDichotomyValidationData> validateNetwork(Network n
                             true);
                 } else {
                     businessLogger.info("Angle monitoring result is unsecure");
+                    crac.getAngleCnecs().forEach(
+                            angleCnec -> {
+                                if (raoResultWithAngleMonitoring.getMargin(crac.getLastInstant(), angleCnec, Unit.DEGREE) < 0) {
+                                    businessLogger.info("Angle {} value's {}", angleCnec.getName(), raoResultWithAngleMonitoring.getAngle(crac.getLastInstant(), angleCnec, Unit.DEGREE));
```

Comment:
Just change the message to match what is given in the ticket:
```java
"Angle {}'s value is {}"
```

Plus is the "degree" unit automatically added in the `getAngle()` response? Or should it be added in the message template as well?